### PR TITLE
Update openapi.md

### PR DESCRIPTION
**Pull request by kpn13**
This PR adds an exemple on how to define OpenAPI doc server url.

### CODE REVIEW

File: core/openapi.md [hunk 1]
```diff
@@ -95,6 +95,9 @@ class OpenApiFactory implements OpenApiFactoryInterface
         $openApi = $openApi->withExtensionProperty('key', 'Custom x-key value');
         $openApi = $openApi->withExtensionProperty('x-value', 'Custom x-value value');
 
+        // to define base path URL
+        $openApi = $openApi->->withServers([new Server('https://foo.bar')]);
```
Comment (alanpoulain) [inline core/openapi.md:99]: **issue:** There is a syntax error. You need to change the sentence before the code block too.

Comment (kpn13) [reply core/openapi.md]: Thx for spotting the error, fixed now. WDYM about the sentence pls?

Comment (kpn13) [reply core/openapi.md]: Oh indeed, thank you, done.

File: core/openapi.md [hunk 2]
```diff
@@ -95,6 +95,9 @@ class OpenApiFactory implements OpenApiFactoryInterface
         $openApi = $openApi->withExtensionProperty('key', 'Custom x-key value');
         $openApi = $openApi->withExtensionProperty('x-value', 'Custom x-value value');
 
+        // to define base path URL
+        $openApi = $openApi->withServers([new Server('https://foo.bar')]);
```
Comment (alanpoulain) [inline core/openapi.md:99]: **issue:** I think you need to add a `use` for `Server` too.

Comment (kpn13) [reply core/openapi.md]: Fixed it by using `Model` ns, thx again :)

**Review by alanpoulain — APPROVED** [review]

**Pull request merged**